doc/CodingStandard.md

```diff
@@ -412,6 +412,17 @@ if (NULL != myPointer) {
 }
 ```
 
+* Utilize the below format for comparisons (<, <=, >, or >=) of the form _min <= value <= max_ to improve clarity.
```

Comment:
Using brackets and a code block will be better in highlighting the notation than italicizing the text.
```suggestion
* Utilize the below format for comparisons `(<, <=, >, or >=)` of the form `(min <= value <= max)` to improve clarity.
```